javatests/arcs/core/entity/DummyEntity.kt

```diff
@@ -59,6 +59,6 @@ class DummyEntity : EntityBase(ENTITY_CLASS_NAME, SCHEMA), Storable {
                 )
             ),
             hash = SCHEMA_HASH
-        )
+        ).also { SchemaRegistry.register(it) }
```

Comment:
I preferred doing this explicitly in tests, rather than relying on it being the default. I think it helps make it more explicit what's going on.

Reply:
Reverted :D